Dockerfile

```
@@ -27,11 +27,17 @@ ARG BUILD_HASH
+RUN rm -f package-lock.json && npm install && npm cache clean --force
+
 
 COPY . .
 ENV APP_BUILD_HASH=${BUILD_HASH}
-RUN npm run build
+ENV NODE_ENV=production
+ENV VITE_DISABLE_SOURCEMAPS=true
+
+RUN npm run pyodide:fetch
+
+RUN node --max-old-space-size=4096 ./node_modules/vite/bin/vite.js build
```

Hadolint (2.12.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)